.github/workflows/push-build-downstream.yaml

```diff
@@ -0,0 +1,56 @@
+name: Push Image Build Downstream
+on:
+  push:
+    branches:
+      - master
+      - release-*
+    tags:
```

Comment:
We can remove the tags I think.
Downstream we are only maintaining branches.

Reply:
Great catch! sure,updated